### ISSUE TITLE
v1.60.0 macOS "log out" UI

### ISSUE THREAD

**Issue by thealanberman** [tailscale/tailscale#11264]
### What is the issue?

macOS 14.3.1 -- just upgraded to 1.60.0 yesterday

Opened up the new settings window, clicked "log out" -- the waiting spinner icon just ran constantly. The icon in the menubar showed logged out, but the "processing" wait indicator didn't seem to acknowledge.

![Screenshot 2024-02-27 at 10 12 23 AM](https://github.com/tailscale/tailscale/assets/796535/d0a32b30-686a-4e2c-bc11-5b572d2e25ff)

### Steps to reproduce

1. log in to tailscale
2. open up the "settings" window
3. go to "accounts"
4. click "log out"

### Are there any recent changes that introduced the issue?

Updated to 1.60.0 from 1.58.2

### OS

macOS

### OS version

14.3.1

### Tailscale version

1.60.0

**Comment by kelivel** [comment]
Can you generate a [bugreport](https://tailscale.com/kb/1227/bug-report) after logout fails and paste the bug marker here?

**Comment by thealanberman** [comment]
BUG-230e77119373cd2e51526ebecdeb6bcabcdc1c1ab13fd85ffe9bba8ce4dc71ab-20240227193503Z-f7ebf614e0180df2

Interestingly, I also got this notification when I click log out.

![Screenshot 2024-02-27 at 1 34 54 PM](https://github.com/tailscale/tailscale/assets/796535/6a39125c-4f3f-41a2-b5d3-79328110c4e3)


And yet, up in the menubar it shows I'm logged out.

![Screenshot 2024-02-27 at 1 35 52 PM](https://github.com/tailscale/tailscale/assets/796535/6316c680-88c4-4681-a065-2bc2517463ee)

**Comment by agottardo** [comment]
@thealanberman can you log out successfully from the CLI? What happens if you log into a second account, does that allow you to log out of the first one?

**Comment by thealanberman** [comment]
I ran

```
/Applications/Tailscale.app/Contents/MacOS/Tailscale logout
```
and the icon in the menubar went gray, indicating I'm logged out... but the
command appears to be hanging in the terminal and doesn't return me to my
prompt.

I was able to log in with a different account.

When I clicked the "Log out" button for that 2nd account I similarly got
the spinning wheel.

I've now tried removing all accounts, restarting the app, and signing in
fresh with just the one (work) account. It let me sign in no problem, but
the "log out" button continues to just result in spinning wheel.

- Al
